source/kraken/make_disruption_from_chaos.cpp

```diff
@@ -251,6 +251,12 @@ static std::set<nt::disruption::ChannelType> create_channel_types(const chaos::C
         case chaos::Channel_Type_unkown_type:
             res.insert(nt::disruption::ChannelType::unknown_type);
             break;
+        case chaos::Channel_Type_title:
+            res.insert(nt::disruption::ChannelType::title);
+            break;
+        case chaos::Channel_Type_beacon:
+            res.insert(nt::disruption::ChannelType::beacon);
+            break;
         default:
             throw navitia::exception("Unhandled ChannelType value in Chaos.Proto");
```

Comment:
we should apply the same rules between disruptions from the database or directly from chaos.
So the default case should put an `Channel_Type_unkown_type`.
Also a log when this happen will be useful